anvil-python/anvil/commands/haproxy.py

```diff
@@ -154,24 +189,33 @@ def prompt(self, console: Console | None = None) -> None:
         variables.setdefault("virtual_ip", "")
         variables.setdefault("ssl_cert", "")
         variables.setdefault("ssl_key", "")
+        variables.setdefault("ssl_cacert", "")
+        variables.setdefault("tls_mode", "disabled")
 
         # Set defaults
         self.preseed.setdefault("virtual_ip", "")
         self.preseed.setdefault("ssl_cert", "")
         self.preseed.setdefault("ssl_key", "")
+        self.preseed.setdefault("ssl_cacert", "")
+        self.preseed.setdefault("tls_mode", "disabled")
 
+        qs = haproxy_questions()
+        qs.update(tls_questions(HAPROXY_VALID_TLS_MODES))
         haproxy_config_bank = questions.QuestionBank(
-            questions=haproxy_questions(),
+            questions=qs,
             console=console,
             preseed=self.preseed.get("haproxy"),
```

Comment:
Your plan is to consume the defaults (preseed) from section `haproxy`. This is fine but you need to put inside preesed both haproxy_questions and also tls_questions. Or you can have two sections. Please take a look at `anvil-python/anvil/provider/local/deployment.py#85`. Whatever you decide, please make sure that the preseed (and manifest) include all fields into one or two sections.